### DISCUSSION TITLE
Using Windicss with different color schemes for light/dark mode

### DISCUSSION THREAD

**Discussion by ImpossibleReality** [windicss/windicss#385] [Q&A]
I am working on migrating my project from css to windi 

Before, using css, I used to do something like this:
```css
html {
  --background-color: white
}
html.dark {
  --background-color: gray
}
```
Currently, I am using windi shortcuts to do the same thing, but I was wondering if there was a less hacky solution to doing this:
```js
// windi.config.js

export default {
  theme: {
    colors: {
      'light-bg': 'white',
      'dark-bg': 'gray',
    },
  },
  shortcuts: {
    'background': 'bg-light-bg dark:bg-dark-bg',
  },
}
```
The problem with my current approach is that, although it may be straightforward for background colors, when I'm working with my primary colors (which have a slightly brighter dark mode color), I may have to make different shortcuts for borders, svg fills, text color, as well as background color. This clutters the config file a ton for something that should be really simple.

You might also say that I should just specify the light and dark colors for each component that I make, but that is even more tedious than specifying 4 different shortcuts.

Am I missing something important?

**Comment by peterbud** [comment]
@ImpossibleReality, have you found any good solution for this?

**Reply by ImpossibleReality** [reply]
Nope, still using WindiCSS Shortcuts 😔
